packages/guides-restructured-text/resources/template/html/body/directive/confval.html.twig

```diff
@@ -0,0 +1,14 @@
+<dl class="confval">
+    <dt id="{{ node.id }}">
+        <code class="sig-name descname"><span class="pre">{{ node.plainContent }}</span></code></dt>
+    <dd>
+        <div class="line-block">
+            <div class="line"><strong>Type:</strong> string</div>
+            <div class="line"><strong>Required:</strong> <code class="docutils literal notranslate"><span class="pre">False</span></code></div>
+            <div class="line"><strong>Default:</strong> <code class="docutils literal notranslate"><span class="pre">"Hello</span> <span class="pre">World"</span></code></div>
```

Comment:
This seem to be fixed values that need an change?

Reply:
duh! true enough